packages/frontend/src/hooks/useFieldValues.ts

```diff
@@ -87,14 +92,23 @@ export const useFieldValues = (
         },
         [filters, initialData],
     );
+    const cachekey = [
+        'project',
+        projectId,
+        tableName,
+        fieldName,
+        'search',
+        debouncedSearch,
```

Comment:
extracted cacheKey in a separate variable, but I didn't change it. We don't need to include forceRefresh here since we'll be using `refetch`
Note: Adding forceRefresh here will trigger 2 queries (one with true, and another with false)